packages/salesforcedx-utils-vscode/src/context/workspaceContextUtil.ts

```diff
@@ -51,6 +51,10 @@ export class WorkspaceContextUtil {
     this.cliConfigWatcher.onDidDelete(bindedHandler);
   }
 
+  public static getLogDirPath(): string {
+    return join(getRootWorkspacePath(), '.sfdx', 'tools', 'debug', 'logs');
```

Comment:
Should the const for SFDX be used here?

Reply:
Ordinarily I'd agree but these will all be updated with the core v3 work so I was cool with leaving it.